src/content/configuration/watch.md

```diff
@@ -70,3 +70,46 @@ poll: 1000 // Check for changes every second
 
 T> If watching does not work for you, try out this option. Watching does not work with NFS and machines in VirtualBox.
 
+# Troubleshooting
+
+## Webpack doesn't recompile on change while watching
+
+### File changes are being seen, just no files are being updated
+
+Verify that webpack is not being notified of changes by running with the --progress flag. If progress shows on save but no files are output, it is likely a configuration issue, not a file watching issue.
```

Comment:
@EugeneHlushko 

I would suggest a small grammar and formatting tweak…

running webpack with the `--progress` flag